crates/webidl/tests/all/simple.rs

```diff
@@ -44,14 +44,14 @@ fn method() {
                     let pi = Foo::new(3.14159).unwrap();
                     let e = Foo::new(2.71828).unwrap();
                     // TODO: figure out why the following doesn't fail
```

Comment:
FWIW this should be fixed now in upstream rust if you wanna do a drive-by cleanup as well